google-cloud-testing/spanner-benchwrapper/src/main/java/com/google/cloud/benchwrapper/SpannerBenchWrapperImpl.java

```diff
@@ -39,13 +41,16 @@
   private Spanner spanner;
   private DatabaseClient dbClient;
 
-  public SpannerBenchWrapperImpl() {
-    SpannerOptions options = SpannerOptions.newBuilder().setProjectId("test-project-id").build();
+  public SpannerBenchWrapperImpl(String spannerEmulatorHost) {
+    final SpannerOptions.Builder optionsBuilder = SpannerOptions.newBuilder();
+    final SpannerOptions options = optionsBuilder
+        .setEmulatorHost(spannerEmulatorHost)
+        .build();
     spanner = options.getService();
 
     dbClient =
         spanner.getDatabaseClient(
-            DatabaseId.of(options.getProjectId(), "test-instance", "test-db"));
+            DatabaseId.of(options.getProjectId(), "someinstance", "somedatabase"));
```

Comment:
nit: I think we mostly use `test-instance` / `test-db` in other samples

Reply:
I might leave this as is for now if that's okay, the other existing client library benchwrappers are all using `someinstance`/`somedatabase` already, so would need to change them all and the google3 scripts if I changed these here.